pywren/wren.py

```diff
@@ -60,7 +59,7 @@ def remote_executor(config=None, job_max_runtime=3600):
 
     AWS_REGION = config['account']['aws_region']
     SQS_QUEUE = config['standalone']['sqs_queue_name']
-    invoker = invokers.SQSInvoker(AWS_REGION, SQS_QUEUE)
+    invoker = queues.SQSInvoker(AWS_REGION, SQS_QUEUE)
 
```

Comment:
Agree that would overkill.
Another way is to wrap SQSInvoker in `invokers.py` (We probably want to change the naming  as well, having a invoker inside a queue.py is not ideal). But this should not be in this PR.
I'm fine with merging.